### PR TITLE
source: obsolete dirbrowser entries  cleanup

### LINKED ISSUE #2005: Ceanup dirbrowser references in source
removed KviInternalCommand **dirbrowser.open** entries in https://github.com/kvirc/KVIrc/pull/2000 since this module no longer exists

related to **dirbrowser** cleanup
todo:
- [x] cleanup other instances of dirbrowser references https://github.com/kvirc/KVIrc/search?utf8=%E2%9C%93&q=dirbrowser.open

### PR DESCRIPTION
**Pull request by un1versal**
#### Changes proposed
- dirbrowser module has been gone for a long time say good bye to remnants 

resolves #2005 

cleaned-up in:
- KviMediaManager.cpp
- KviWindow.h/.cpp

Tagged for attention from dev due to entries in https://github.com/un1versal/KVIrc/blob/633afc75164bbe272ef1897d4b5854ef12209368/src/kvilib/ext/KviMediaManager.cpp#L231-L297

Ide like to know what they do and when they are shown or if they are even used currently or was this all part of same dirbrowser?

### CODE REVIEW

**Comment by un1versal** [comment]
No one knows about these at all?

File: src/kvirc/ui/KviWindow.cpp [hunk 1]
```diff
@@ -320,14 +320,13 @@ const char * KviWindow::m_typeTable[TypeCount] = {
 	"userwindow",   // 16
 	"tool",         // 17
 	"iograph",      // 18
-	"dirbrowser",   // 19
```
Comment (staticfox) [inline src/kvirc/ui/KviWindow.cpp:328]: Please don't delete this and move down the names, it will mess up the order of the array since the names are in a specific order (similar to the config ordeal)

File: src/kvirc/ui/KviWindow.h [hunk 2]
```diff
@@ -148,13 +148,12 @@ class KVIRC_API KviWindow : public QWidget
...
-		Offer = 23,
-		Debug = 24,
-		Unknown = 25,
+		ScriptEditor = 19,
+		ScriptObject = 20,
+		LogView = 21,
+		Offer = 22,
+		Debug = 23,
+		Unknown = 24,
 		TypeCount = 26
```
Comment (staticfox) [inline src/kvirc/ui/KviWindow.h:158]: Similar to above, however you didn't update the typeCount so the array has a `NULL` at the end.

File: src/kvirc/ui/KviWindow.cpp (hunk 1 above)
Comment (un1versal) [inline src/kvirc/ui/KviWindow.cpp:328]: I dont think its the same but at this stage Ive resigned to the fact we like trash so Ill comment them out

File: src/kvirc/ui/KviWindow.h (hunk 2 above)
Comment (un1versal) [inline src/kvirc/ui/KviWindow.h:158]: My local has count 25, I guess that would be right then. but since we have to comment these out Ill revert and leave the trash behind

**Comment by un1versal** [comment]
@staticfox would you know if this is used https://github.com/un1versal/KVIrc/blob/633afc75164bbe272ef1897d4b5854ef12209368/src/kvilib/ext/KviMediaManager.cpp#L231-L297

**Comment by staticfox** [comment]
Looks like it's still used, I'm not sure how to invoke it though.

**Comment by un1versal** [comment]
> Looks like it's still used, I'm not sure how to invoke it though.

Addressed comments, thx for review, if its OK please merge.

File: src/kvirc/ui/KviWindow.cpp [hunk 3]
```diff
@@ -324,15 +324,14 @@ const char * KviWindow::m_typeTable[TypeCount] = {
 	"dccvideo",     // 15
 	"userwindow",   // 16
 	"tool",         // 17
-	"iograph",      // 18
```
Comment (staticfox) [inline src/kvirc/ui/KviWindow.cpp:327]: why did you delete iograph?

File: src/kvirc/ui/KviWindow.h [hunk 4]
```diff
@@ -148,14 +148,13 @@ class KVIRC_API KviWindow : public QWidget
...
 		IOGraph = 18,
-		DirBrowser = 19,
+		DirBrowser = 19,   //!\ no longer exists please reuse entry
 		ScriptEditor = 20,
 		ScriptObject = 21,
 		LogView = 22,
 		Offer = 23,
 		Debug = 24,
 		Unknown = 25,
-		TypeCount = 26
```
Comment (staticfox) [inline src/kvirc/ui/KviWindow.h:158]: And this?

Comment (un1versal) [inline src/kvirc/ui/KviWindow.h:158]: roonk.... yea sec...

File: src/kvirc/ui/KviWindow.cpp (hunk 3 above)
Comment (un1versal) [inline src/kvirc/ui/KviWindow.cpp:327]: Not on purpose, c&p monster strikes again.

cookiemonster

**Comment by un1versal** [comment]
@staticfox done, should be it.

**Comment by staticfox** [comment]
added in https://github.com/kvirc/KVIrc/commit/80370cb6f97b89de9bc30b1a2d63157ef54e8885

**Comment by un1versal** [comment]
> added in 80370cb

this was in sync a merge was possible, please avoid closing prs when they can be merged.
